js/scripts.js

```diff
@@ -9,22 +9,26 @@ $(function() {
   };
 
   function createTweet(input) {
-    var data = input[0];
-    var quoteText = $(data.content).text().trim();
-    var quoteAuthor = data.title;
-    var tweetText = 'Quote of the day - ' + quoteText + ' Author: ' + quoteAuthor;
-  
-    if (!quoteAuthor.length) {
-      quoteAuthor = 'Unknown author...';
-    };
+    if (!(input.length === 0)) {
```

Comment:
how to write it easier? try to write that if conditional isn't in if conditional :)